### ISSUE TITLE
System.Drawing.Common Dependency still exist in EEPlus 6 when using nuget

### ISSUE THREAD

**Issue by shen3221894** [EPPlusSoftware/EPPlus#634]
For some reson, System.Drawing.Common.dll is not allowed in project. When installing EPPlus.6.0.3 by nuget, System.Drawing.Common Dependency still exist as follow. How to remove the Dependency completely?

Installing:
EPPlus.6.0.3
EPPlus.Interfaces.6.0.0
EPPlus.System.Drawing.6.0.0
Microsoft.Extensions.Configuration.5.0.0
Microsoft.Extensions.Configuration.Abstractions.5.0.0
Microsoft.Extensions.Configuration.FileExtensions.5.0.0
Microsoft.Extensions.Configuration.Json.5.0.0
Microsoft.Extensions.FileProviders.Abstractions.5.0.0
Microsoft.Extensions.FileProviders.Physical.5.0.0
Microsoft.Extensions.FileSystemGlobbing.5.0.0
Microsoft.Extensions.Primitives.5.0.0
Microsoft.IO.RecyclableMemoryStream.2.2.0
Microsoft.NETCore.Platforms.5.0.0
Microsoft.Win32.SystemEvents.5.0.0
**System.Drawing.Common.5.0.0**
System.Formats.Asn1.5.0.0
System.Security.Cryptography.Cng.5.0.0
System.Security.Cryptography.Pkcs.5.0.0
System.Text.Encoding.CodePages.5.0.0

**Comment by JanKallman** [comment]
Yes, it's referenced by EPPlus.System.Drawing, but it's not used if you don't have access to GDI. What do you mean with that it's not allowed?

**Comment by swmal** [comment]
the commit above is referencing this issue by mistake, it should reference #632

**Comment by shen3221894** [comment]
> Yes, it's referenced by EPPlus.System.Drawing, but it's not used if you don't have access to GDI. What do you mean with that it's not allowed?

There is a plan to use EEPlus 6 into a Unity3d IL2CPP project for us. But
1. For the API with .Net Framework 4.x, IL2CPP build failure for 'System.NotSupportedException: System.Configuration.ConfigurationManager::get_AppSettings'
2. For the API with .Net Standard 2.1, IL2CPP build failure for System.Drawing.Common.dll compatibility

So, is there a method for avoiding the errors above?

**Comment by JanKallman** [comment]
Hmm, I haven't used Unity3D IL2CPP.
Does it perform the compatibility tests on build? If so, there is a problem as EPPlus references System.Drawing.Comon via the EPPlus.System.Drawing reference. However, it does not use it if the environment does not support GDI. Is it possible to configure it to do this check in runtime instead? Optionally, I assume we could create a minimum dependency Nuget package without the reference to EPPlus.System.Drawing.

**Comment by shen3221894** [comment]
Yes, I think the compatibility tests of reference is performed on build. I try to remove the System.Configuration reference and related code about 'System.Configuration.ConfigurationManager' from souce code, and it works now! 
Thanks a lot!

**Comment by elfasito** [comment]
Hi, Im using EPPLUS 6.0.6 from nuget.
I have the same Issue with: System.Configuration.ConfigurationManager (unity il2cpp)
in app run I get: 
System.NotSupportedException: System.Configuration.ConfigurationManager::get_AppSettings

Im novice so I dont have a clue how to solve it, like @shen3221894  do.
Can someone give me some guidance?.
